pkg/api/meta/meta_test.go

```diff
@@ -739,6 +739,8 @@ type MyAPIObject2 struct {
 func getObjectMetaAndOwnerRefereneces() (myAPIObject2 MyAPIObject2, metaOwnerReferences []metatypes.OwnerReference) {
```

Comment:
Actually this is not the latest commit I pushed. I don't know why github is showing the stale commit.